Sharding Testgrid Config: sig-network, sig-node

chases2 opened the pull request:
xref #13541
Owners inherited from //config/jobs:
/cc @bowei @cadmuxe @jingax10 @rramkumar1 @MrHohn 
/cc @yujuhong @Random-Liu @dchen1107

bowei commented on config/testgrids/kubernetes/sig-network/OWNERS:
```diff
@@ -0,0 +1,8 @@
+approvers:
+- bowei
+- cadmuxe
+- jingax10
```
jing can be remove (no longer with project)

chases2 commented:
- Fixed merge conflicts
- Removed @cadmuxe; once they've been added as a Kubernetes org member, they can be added again
- Removed @jingax10 per comment
